sample/ApplicationInsightsHostingStartupSample/ApplicationInsightsHostingStartupSample.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
please move to `test` folder if this app only needed for integration tests. Also consider re-use one of apps that is already there